exercises/pangram/pangram_test.rb

```diff
@@ -19,6 +19,16 @@ def test_pangram_with_only_lower_case
     assert Pangram.is_pangram?(str)
   end
 
+  def test_pangram_with_underscore_character
+    str = 'the quick brown_fox jumps over the lazy dog'
+    assert Pangram.is_pangram?(str)
+  end
+
+  def test_pangram_with_sentence_containing_numbers
+    str = 'the qu1ck br0wn fox jumps over the lazy dog'
```

Comment:
Avoid 0 and 1 as they are too close in some fonts to l and O.  When possible, use distinctively shaped numbers, rather than the possibly visually ambiguous ones.

This becomes aggravated especially when used in places where those ambiguities are replacements of letters that are so similar.